web/README.md

```diff
@@ -95,8 +96,24 @@ In general all global atoms should be accessed through useAtom and not through p
 If the components are very closely related it may make sense to pass down derived values instead.
 
 - ButtonGroup.tsx
-- Button1.tsx
-- Button2.tsx
-- Button3.tsx
+  - Button1.tsx
+  - Button2.tsx
+  - Button3.tsx
```

Comment:
Having the buttons on the same level as the button groups, does not really convey that the group is distributing data to the buttons.